galaxy/Dockerfile

```diff
@@ -96,6 +96,7 @@ RUN mkdir $GALAXY_ROOT/tool_deps
 RUN . $GALAXY_VIRTUALENV/bin/activate && python scripts/fetch_eggs.py -c $GALAXY_CONFIG_FILE && python scripts/fetch_eggs.py -e drmaa -c $GALAXY_CONFIG_FILE
 # Installing latest version of setuptools
 RUN . $GALAXY_VIRTUALENV/bin/activate && pip install setuptools --upgrade
+RUN . $GALAXY_VIRTUALENV/bin/activate && pip install bioblend
```

Comment:
@cameron-jack I don't think this is needed as Galaxy has already a dependency on bioblend: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/dependencies/pinned-requirements.txt#L34